client/my-sites/posts/post-controls.jsx

```diff
@@ -144,14 +145,14 @@ const getAvailableControls = props => {
 
 	if ( controls.more.length ) {
 		controls.main.push( {
-			className: 'more',
+			className: 'more' + ( current === 'more' ? ' is-active' : '' ),
 			icon: 'ellipsis',
 			onClick: onShowMore,
 			text: translate( 'More' ),
 		} );
 
 		controls.more.push( {
-			className: 'back',
+			className: 'back' + ( current === 'back' ? ' is-active' : '' ),
```

Comment:
Thinking well it will need to contemplate some exceptions because first, there is not a control `name` property and we can't trust on the className value since there are some controls that have more that one class.